src/commandExecution.ts

```diff
@@ -183,6 +203,10 @@ export class CommandExecution extends AsyncCreatable {
         } else if (argv.find((arg) => new RegExp(`^--${flagName}(=.*)?$`).test(arg))) {
           this.specifiedFlags.push(flagName);
           this.specifiedFlagFullNames.push(flagName);
+        } else if (flagDefinitions[flagName].deprecateAliases) {
+          // we can't find the flag as the key (long name) or short char, so it must be a deprecated flag
+          const argvFlags = this.argv.map((a) => a.match(/-(\w+)/g)).map((a) => a?.[0].replace('-', ''));
```

Comment:
`/-(\w+)/g`: this will also catch alphanumeric chars but `flag.char` can only be a lower/upper case alphabet char:
https://github.com/oclif/core/blob/3c8cc06946b11f53006f4a58cd521eb33fd102a5/src/interfaces/parser.ts#L162

I've looked at our stuff and most `Flags\.duration` has `min` set to ensure a positive value, but it's something that could happen with 3rd party commands:
https://github.com/search?q=org%3Asalesforcecli%20%2FFlags%5C.duration%2F&type=code

Reply:
🕵️ you're right! I'll update that 🙇